setup.py

```diff
@@ -1,27 +1,12 @@
+from distutils.core import setup
+import setuptools
 import os
-from setuptools import setup, find_packages
+from os.path import join
```

Comment:
I do not see any use of this import. If that is the case, should we remove it?

Reply:
c18b02f